app/src/main/java/com/example/textreccy/MainActivity.kt

```diff
@@ -50,11 +46,7 @@ class MainActivity : AppCompatActivity() {
         super.onActivityResult(requestCode, resultCode, data)
         if (requestCode == REQUEST_CAMERA_CAPTURE && resultCode == RESULT_OK){
 
-//            lateinit var mCaptureRequestBuilder : Builder
```

Comment:
We need to get the focus of the camera working.

Reply:
I agree, we'll do this on a separate pull request.

Reply:
No